server/plugin/template.go

```diff
@@ -149,10 +169,24 @@ func init() {
 		`{{template "eventRepoIssueFullLink" .}} - {{.GetIssue.GetTitle}}`,
 	))
 
+	template.Must(masterTemplate.New("labels").Funcs(funcMap).Parse(`
+{{- if .Labels }}
+Labels: {{range $i, $el := .Labels -}}` + "{{- if $i}}, {{end}}[`{{ $el.Name }}`]({{ $.RepositoryURL }}/labels/{{ $el.Name | pathEscape }})" + `{{end -}}
+{{ end -}}
+`))
+
+	template.Must(masterTemplate.New("assignee").Funcs(funcMap).Parse(`
+{{- if .Assignees }}
+Assignees: {{range $i, $el := .Assignees -}} {{- if $i}}, {{end}}[{{ $el.GetLogin }}]({{ $el.GetHTMLURL }}){{end -}}
```

Comment:
If the user has a connected GitHub account, should this link to the Mattermost account instead?

Reply:
Right, I see that you are using such a convention in other places where the user is displayed. I think it makes sense to change it so it is consistent unless anyone has some objections?

Reply:
Makes sense :+1: